app/scripts/modules/kubernetes/src/v2/manifest/status/status.component.ts

```diff
@@ -0,0 +1,23 @@
+import { IComponentOptions, IController, module } from 'angular';
+
+import { IManifestStatus } from '@spinnaker/core';
+
+class KubernetesManifestStatusCtrl implements IController {
+  public status: IManifestStatus;
+}
+
+class KubernetesManifestStatusComponent implements IComponentOptions {
+  public bindings: any = { status: '=' };
```

Comment:
Use `<` instead of `=` if you know there won't be any changes made to the bound property from inside the component. (It's better for performance, and some would say always use `<`).

Reply:
![brain](https://user-images.githubusercontent.com/4874941/32954730-ec510aae-cb81-11e7-8a38-2e62f2d7e191.jpg)